bin/fetch-books.js

```
@@ -1,50 +1,62 @@
 import whilst from 'async/whilst';
-import { recordTypes } from '@grodno-city/alis-web-request';
+import { getRecordByID } from '@grodno-city/alis-web-request';
 import fs from 'fs';
-import { log, alisEndpoint, index } from '../config';
-import { indexRecordsByQuery, collectRequestInfo } from '../index';
+import { log, alisEndpoint, index, client } from '../config';
+import { collectRequestInfo } from '../index';
 
 const snapshot = './bin/.fetch-books-snapshot';
-let year;
+let id = 0;
+let count = 0;
 try {
-  year = fs.readFileSync(snapshot, 'utf8').substr(0, 4);
+  const args = fs.readFileSync(snapshot, 'utf8').split(' ');
+  id = Number(args[0]) + 1;
+  count = Number(args[1]);
 }
 catch (err) {
-  year = (new Date()).getFullYear();
+  id = 0;
```

This `catch` silently ignores errors. Let's replace try/catch block with `if (file exits)` case, because it shows intent more clearly.